man/fi_pingpong.1.md

```diff
@@ -125,31 +143,36 @@ Specifically, this will run a pingpong test with:
 
 # DEFAULTS
 
-There is no default provider; if a provider is not specified via the `-p` switch, the test will pick one from the list of available providers (as returned by `fi_getinfo`(3)).
+There is no default provider; if a provider is not specified via the `-p`
+switch, the test will pick one from the list of available providers (as
+returned by fi_getinfo(3)).
 
 If no endpoint type is specified, 'dgram' is used.
 
-The default tested sizes are:  64, 256, 1024, 4096.
-
-If no server address is specified, the server address is determined by the selected provider. With the current implementation of libfabric, it means that the picked address will be the first address in the list of available addresses matching the selected provider.
-
+The default tested sizes are:  64, 256, 1024, 4096, 655616, and 1048576. The
```

Comment:
> 655616 is a weird size, is it a typo?

You know, I just copied that from the source for the test. It is a typo though, that should be `2 << 15` or `65536`. I'll fix that and also update the test.